opensrp-chw-referral/src/main/java/org/smartregister/chw/referral/activity/BaseIssueReferralActivity.java

```diff
@@ -0,0 +1,363 @@
+package org.smartregister.chw.referral.activity;
+
+import android.annotation.SuppressLint;
+import android.arch.lifecycle.ViewModelProviders;
+import android.content.Context;
+import android.database.Cursor;
+import android.databinding.DataBindingUtil;
+import android.databinding.Observable;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.Toolbar;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.AutoCompleteTextView;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.google.gson.Gson;
+import com.rengwuxian.materialedittext.MaterialEditText;
+import com.wdullaer.materialdatetimepicker.date.DatePickerDialog;
+
+import org.joda.time.DateTime;
+import org.joda.time.Period;
+import org.json.JSONObject;
+import org.smartregister.AllConstants;
+import org.smartregister.chw.referral.R;
+import org.smartregister.chw.referral.ReferralLibrary;
+import org.smartregister.chw.referral.contract.BaseIssueReferralContract;
+import org.smartregister.chw.referral.domain.MemberObject;
+import org.smartregister.chw.referral.domain.ReferralServiceIndicatorObject;
+import org.smartregister.chw.referral.interactor.BaseIssueReferralInteractor;
+import org.smartregister.chw.referral.model.AbstractIssueReferralModel;
+import org.smartregister.chw.referral.model.BaseIssueReferralModel;
+import org.smartregister.chw.referral.presenter.BaseIssueReferralPresenter;
+import org.smartregister.chw.referral.provider.ReferralServicesProvider;
+import org.smartregister.chw.referral.util.Constants;
+import org.smartregister.commonregistry.CommonPersonObject;
+import org.smartregister.commonregistry.CommonPersonObjectClient;
+import org.smartregister.commonregistry.CommonRepository;
+import org.smartregister.domain.Location;
+import org.smartregister.chw.referral.databinding.ActivityReferralRegistrationBinding;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+import fr.ganfra.materialspinner.MaterialSpinner;
+import timber.log.Timber;
+
+/**
+ * Created by cozej4 on 2019-10-07.
+ *
+ * @cozej4 https://github.com/cozej4
+ */
+
+public class BaseIssueReferralActivity extends AppCompatActivity implements BaseIssueReferralContract.View {
+    protected BaseIssueReferralContract.Presenter presenter;
+    protected String BASE_ENTITY_ID;
+    protected List<String> SERVICE_IDS;
+    protected String ACTION;
+    protected String FORM_NAME;
```

Comment:
Any reason for naming these fields like constants? I guess using camel case makes it more readable and maintains the convention.

Reply:
Noted